### PR TITLE
Filter-function examples

### PR DESCRIPTION
**Pull request by mfluehr**

### CODE REVIEW

File: live-examples/css-examples/filter-effects/grayscale.html [hunk 1]
```diff
@@ -0,0 +1,35 @@
+<section id="example-choice-list" class="example-choice-list large" data-property="filter">
+<div class="example-choice">
+<pre><code id="example_one" class="language-css">filter: grayscale(0);</code></pre>
+<button type="button" class="copy hidden" aria-hidden="true">
+    <span class="visually-hidden">Copy to Clipboard</span>
+</button>
+</div>
+
+<div class="example-choice" initial-choice="true">
+<pre><code id="example_one" class="language-css">filter: grayscale(0.20);</code></pre>
```
Comment (wbamberg) [inline live-examples/css-examples/filter-effects/grayscale.html]: "example_one" -> "example_two" (and the subsequent examples need to increment as well of course).

File: live-examples/css-examples/filter-effects/hue-rotate.html [hunk 2]
```diff
@@ -0,0 +1,35 @@
+<section id="example-choice-list" class="example-choice-list large" data-property="filter">
+<div class="example-choice">
+<pre><code id="example_one" class="language-css">filter: hue-rotate(0);</code></pre>
+<button type="button" class="copy hidden" aria-hidden="true">
+    <span class="visually-hidden">Copy to Clipboard</span>
+</button>
+</div>
+
+<div class="example-choice" initial-choice="true">
+<pre><code id="example_one" class="language-css">filter: hue-rotate(90deg);</code></pre>
```
Comment (wbamberg) [inline live-examples/css-examples/filter-effects/hue-rotate.html]: Again here, "example_one" -> "example_two" (and the subsequent examples need to increment as well).

File: live-examples/css-examples/filter-effects/invert.html [hunk 3]
```diff
@@ -0,0 +1,42 @@
+<section id="example-choice-list" class="example-choice-list large" data-property="filter">
+<div class="example-choice">
+<pre><code id="example_one" class="language-css">filter: invert(0);</code></pre>
+<button type="button" class="copy hidden" aria-hidden="true">
+    <span class="visually-hidden">Copy to Clipboard</span>
+</button>
+</div>
+
+<div class="example-choice" initial-choice="true">
+<pre><code id="example_one" class="language-css">filter: invert(0.30);</code></pre>
```
Comment (wbamberg) [inline live-examples/css-examples/filter-effects/invert.html]: ...and again...

File: live-examples/css-examples/filter-effects/opacity.html [hunk 4]
```diff
@@ -0,0 +1,42 @@
+<section id="example-choice-list" class="example-choice-list large" data-property="filter">
+<div class="example-choice">
+<pre><code id="example_one" class="language-css">filter: opacity(1);</code></pre>
+<button type="button" class="copy hidden" aria-hidden="true">
+    <span class="visually-hidden">Copy to Clipboard</span>
+</button>
+</div>
+
+<div class="example-choice" initial-choice="true">
+<pre><code id="example_one" class="language-css">filter: opacity(80%);</code></pre>
```
Comment (wbamberg) [inline live-examples/css-examples/filter-effects/opacity.html]: ...and again...

File: live-examples/css-examples/filter-effects/sepia.html [hunk 5]
```diff
@@ -1,20 +1,27 @@
...
+<div class="example-choice">
+<pre><code id="example_one" class="language-css">filter: sepia(0);</code></pre>
+<button type="button" class="copy hidden" aria-hidden="true">
+    <span class="visually-hidden">Copy to Clipboard</span>
+</button>
+</div>
+
 <div class="example-choice" initial-choice="true">
-<pre><code id="example_one" class="language-css">filter: sepia(60%);</code></pre>
+<pre><code id="example_one" class="language-css">filter: sepia(0.20);</code></pre>
```
Comment (wbamberg) [inline live-examples/css-examples/filter-effects/sepia.html]: ...and one more. We should try to generate these IDs, too, it is easy to get them wrong.

File: live-examples/css-examples/filter-effects/sepia.html [hunk 6]
```diff
@@ -1,20 +1,27 @@
...
-<pre><code id="example_two" class="language-css">filter: sepia(0.20);</code></pre>
+<pre><code id="example_two" class="language-css">filter: sepia(60%);</code></pre>
 <button type="button" class="copy hidden" aria-hidden="true">
     <span class="visually-hidden">Copy to Clipboard</span>
 </button>
 </div>
 
 <div class="example-choice">
-<pre><code id="example_three" class="language-css">filter: sepia(0.80);</code></pre>
+<pre><code id="example_three" class="language-css">filter: sepia(1);</code></pre>
```
Comment (wbamberg) [inline live-examples/css-examples/filter-effects/sepia.html]: I like these changes to sepia()

File: live-examples/css-examples/filter-effects/meta.json [hunk 7]
```diff
@@ -8,6 +8,78 @@
             "title": "CSS Demo: filter",
             "type": "css"
         },
+        "blur": {
```
Comment (wbamberg) [inline live-examples/css-examples/filter-effects/meta.json]: should we keep alphabetical order here and insert `filter` in among the functions, or did you keep it first on purpose? I'm not sure what is better, your call.

**Comment by mfluehr** [comment]
Ok, I got the numbering corrected. I think you're right that the "meta" files should always be purely alphabetical--removes any subjectivity to it.

File: live-examples/css-examples/filter-effects/sepia.html (hunk 5 above)
Comment (danielhickman) [reply live-examples/css-examples/filter-effects/sepia.html]: We could [lint for duplicate ids](https://github.com/htmllint/htmllint/wiki/Options#id-no-dup) in with TravisCI to *reduce* the number of issues in the future.

Comment (danielhickman) [reply live-examples/css-examples/filter-effects/sepia.html]: Actually, does the editor even use IDs? That's probably something I should've check when removing clipboard targets but I assumed they were used elsewhere. Removing them doesn't seem to change anything and I can't find anything in the code.

Comment (wbamberg) [reply live-examples/css-examples/filter-effects/sepia.html]: I don't know :). @schalkneethling probably does though.

**Pull request merged**